tests/servers/good.rb

```diff
@@ -333,5 +333,6 @@ def split_header_value(str)
 
 EM.run do
   EM.start_server("127.0.0.1", 9292, GoodServer)
+  EM.start_server("::1", 9293, GoodServer)
```

Comment:
Thanks @kbrock! 

I did notice that issue in my search yesterday.  I tried the latest eventmachine locally but still had the same problem.  I'll re-read the above issue and see if anything jumps out.  I was also going to try jruby 9000 to see if that also fails.